integration-tests/wasm/rust/build.sh

```diff
@@ -0,0 +1,25 @@
+#! /bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -euxo pipefail
+
+rustup target add wasm32-unknown-unknown
+
+cargo build --target wasm32-unknown-unknown --release
```

Comment:
what is about `wasm32-unknown-unknown`? Should user change it on different arch platform?

Reply:
It basically means use a 32Bit address space and compile & execute on any target (I think).

I tested on Mac OS (aarch64) and Linux (x86_64) and it's working fine. The CI build will tell us the state of things on Windows.

Reply:
The Windows job passed.